protocol/daemons/pricefeed/metrics/market_pairs.go

```diff
@@ -45,3 +45,11 @@ func GetMarketPairForTelemetry(marketId types.MarketId) string {
 
 	return marketPair
 }
+
+// ClearMarketPairsForTelemetry clears the in-memory map of marketId to marketPair strings.
+// Used for testing purposes.
+func ClearMarketPairsForTelemetry() {
```

Comment:
Nit: seems that this function is only for testing purposes. Would it be possible to move this to `client_test.go` or make sure `marketToPair` is empty in the test some other way?

Reply:
Good point! Since `marketToPair` is private, iiuc I can either:

1. Move the test into metrics_test.go + change `package metrics_test → package metrics` so it can access the map.
2. Use `unsafe` + `reflect` + `//go:linkname` to access `marketToPair` from a different package.

I'll go with (1) unless you prefer something else